components/layout/fragment.rs

```diff
@@ -1644,7 +1644,7 @@ impl Fragment {
             _   => return None,
         };
 
-        let mut remaining_inline_size = max_inline_size;
+        let mut remaining_inline_size = max_inline_size - self.border_padding.inline_start_end();
```

Comment:
Could you add a comment saying that `self.border_padding.inline_start_end` (presumably) will only be nonzero in the incremental reflow case (I guess?)

This could easily regress.